hometask-2/get_middle_charachter.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+def get_middle(str)
+  if (str.length % 2).zero?
+    puts(str.slice((str.length / 2 - 1)..(str.length / 2)))
```

Comment:
`str.length / 2` appears in your code several times, so how about extracting it into a local variable?